src/internal/templates/internal/lore_wiki.html

```diff
@@ -0,0 +1,72 @@
+{% extends 'internal/base.html' %}
+{% load i18n %}
+{% load static %}
+
+
+{% block title %}
+    {% translate "The wiki of MAKE lore" %}
+{% endblock title %}
+
+{% block header_links %}
+    <link rel="stylesheet" href="{% static 'internal/css/make_lore.css' %}">
+    <link rel="stylesheet" href="{% static 'web/css/header.css' %}"/>
+    <script defer src="{% static 'web/js/header.js' %}"></script>
+{% endblock header_links %}
+
+{% block body %}
+    <div id="lore-page">
+        <div id="lore-wiki-list">
+            <h2>{% translate "Topics" %}
+                {% if perms.internal.add_lore %}
+                    <a href="{% url 'add_lore' %}">
+                        <i class="green plus icon"></i>
+                    </a>
+                {% endif %}
+            </h2>
+            {% include 'internal/includes/lore_articles_list.html' %}
+        </div>
+
+        {# Checkbox belongs to hamburger menu. Checkbox status decides if hamburger menu will be shown or not #}
+        <input type="checkbox" id="burger-input">
+        <label id="lore-burger-icon" for="burger-input"></label>
+        <h2>{% translate "Topics" %}
+            {% if perms.internal.add_lore %}
+                <a href="{% url 'add_lore' %}">
+                    <i class="green plus icon"></i>
+                </a>
+            {% endif %}
+        </h2>
+        <div id="lore-burger-menu">
+            {% include 'internal/includes/lore_articles_list.html' %}
+        </div>
+
+        <div id="lore-article">
+            {% if show_article %}
+                <h1 class="lore-heading">{{ lore_article.title|capfirst }}</h1>
+                {% if perms.internal.delete_lore and lore_article.title != "Dev" %}
+                    <a class="delete-modal-button"
+                        data-url="{% url 'delete_lore' lore_article.slug %}"
+                        data-obj-name="{{ lore_article }}">
+                        <i class="red trash icon"></i>
+                    </a>
+                {% endif %}
+                {% if perms.internal.change_lore and lore_article.title != "Dev" %}
+                    <a href="{% url 'update_lore' lore_article.slug %}">
+                        <i class="make-col-yellow pencil icon"></i>
+                    </a>
+                {% endif %}
+                <div class="lore-container">
+                    {% if lore_article.image %}
+                        <img class="lore-img" src="{{ lore_article.image.url }}" alt="{{ lore_article.title }}">
+                    {% endif %}
+                    <p>{{ lore_article.text|linebreaks }}</p>
+                </div>
+            {% else %}
+                <h1>{% translate "Welcome to the wiki of MAKE lore" %}</h1>
```

Comment:
`<h1>` should not be used here when the headings above are `<h2>`, as this illogical heading hierarchy makes the page less accessible for users with screen readers. Instead, make sure that all the heading levels (1-6) are in logical order, and then style them using either CSS rules or [the `ui <size> header` classes](https://fomantic-ui.com/elements/header.html#content-headers), where `<size>` ranges from `tiny` to `huge` (see the linked docs).

Reply:
I don't really understand the problem here. This HTML file is used as a template by two different views - the general list view that users land on when they first access the lore page, and the detail view that shows a particular lore article. I use The <h1> heading commented on here, is the general heading used by the list view. When a lore article is selected, the heading for the article is also displayed in an <h1> tag.

Reply:
Ah yes, I didn't run the code before reviewing, so I might have spoken too soon here 🤔 I don't have a huge amount of experience with web accessibility, but judging by the page's accessibility tree view (opened in Chrome through the button in the screenshot below), the hierarchy looks okay, I think 🙂
> ![Screenshot of button for opening Google Chrome's accessibility tree view](https://github.com/MAKENTNU/web/assets/6058745/c3c83df3-248b-42e9-b3fe-e173ae7dd760)

Though I guess it could be slightly improved by placing the sidebar and mobile menu elements inside a [`<nav>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav), as their main purpose is to provide navigation between the lore articles, and not as a part of the page's main contents 🤔 In that case, an `aria-label`/`aria-labelledby` attribute should be added to the `<nav>` element, e.g. `aria-label="{% translate "Lore articles" %}"`.